spec/custom_configuration_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+RSpec.describe PipefyMessage::BrokerConfiguration::AwsProvider::ProviderConfig do
+  before do
```

Comment:
I can see that some files import an env inside a block "before", my question is if possible to use an import globally, to don't need to do that, like an `env.test`, WDYT?

Reply:
I think it's possible but the idea here is to have the needed env vars per test if you take a look at the other tests you will see that in most of the cases they don't use the same set of env vars.

But I agree that if we have a lot of repetition of the same thing, in the future, centralizing them at someplace will be really nice 😄